manage-data/lifecycle/index-lifecycle-management/rollover.md

```diff
@@ -9,50 +9,91 @@ products:
 
-* Optimize the active index for high ingest rates on high-performance *hot* nodes.
-* Optimize for search performance on *warm* nodes.
-* Shift older, less frequently accessed data to less expensive *cold* nodes,
-* Delete data according to your retention policies by removing entire indices.
+Without rollover, a single index can grow until searches slow down, shard relocation takes hours, and nodes run low on disk space.
 
-We recommend using [data streams](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-create-data-stream) to manage time series data. Data streams automatically track the write index while keeping configuration to a minimum.
+The rollover feature is an important part of how [index lifecycle](../index-lifecycle-management/index-lifecycle.md) and [data stream lifecycles](../data-stream.md) work to keep your indices fast and manageable. By switching the write target of an index, the rollover action provides the following benefits:
 
-Each data stream requires an [index template](../../data-store/templates.md) that contains:
+* **Automation** - works with ILM to remove manual index rotation tasks and allows for granular control over retention cycles
+* **Optimized performance** - keeps shard sizes within recommended limits (10-50 GB)
+* **Queries run faster** - improves search performance
 
-* A name or wildcard (`*`) pattern for the data stream.
-* The data stream’s timestamp field. This field must be mapped as a [`date`](elasticsearch://reference/elasticsearch/mapping-reference/date.md) or [`date_nanos`](elasticsearch://reference/elasticsearch/mapping-reference/date_nanos.md) field data type and must be included in every document indexed to the data stream.
-* The mappings and settings applied to each backing index when it’s created.
+Rollover can be triggered via the [API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-rollover) or ILM.
 
-Data streams are designed for append-only data, where the data stream name can be used as the operations (read, write, rollover, shrink etc.) target. If your use case requires data to be updated in place, you can instead manage your time series data using [index aliases](../../data-store/aliases.md). However, there are a few more configuration steps and concepts:
+## How rollover works in ILM
 
-* An *index template* that specifies the settings for each new index in the series. You optimize this configuration for ingestion, typically using as many shards as you have hot nodes.
-* An *index alias* that references the entire set of indices.
-* A single index designated as the *write index*. This is the active index that handles all write requests. On each rollover, the new index becomes the write index.
+You define a rollover action in the hot phase of an index lifecycle policy. It will run when any of the configured thresholds are met.
```

Comment:
```suggestion
You define a rollover action in the hot phase of an index lifecycle policy. It will run when any of the configured thresholds are met and the write index contains at least one document.
```